test/test-ciphers-from-pod.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -o errexit
```

Comment:
Can't use 

```
set -o errexit
```
As some of the test requires an error. I'm not sure if there is a better approach.